### PR TITLE
Added formula for Apache Drill

### PR DESCRIPTION
**Pull request by JGailor**

### CODE REVIEW

File: Library/Formula/apache-drill.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Documentation: https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/Formula-Cookbook.md
+#                /usr/local/Library/Contributions/example-formula.rb
+# PLEASE REMOVE ALL GENERATED COMMENTS BEFORE SUBMITTING YOUR PULL REQUEST!
```
Comment (DomT4) [inline Library/Formula/apache-drill.rb]: > PLEASE REMOVE ALL GENERATED COMMENTS BEFORE SUBMITTING YOUR PULL REQUEST!

:wink:

File: Library/Formula/apache-drill.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+class ApacheDrill < Formula
+  homepage ""
```
Comment (DomT4) [inline Library/Formula/apache-drill.rb]: This can be `https://drill.apache.org/download/`.

File: Library/Formula/apache-drill.rb [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  url "http://getdrill.org/drill/download/apache-drill-0.9.0.tar.gz"
```
Comment (DomT4) [inline Library/Formula/apache-drill.rb]: This should be an Apache dyn link ideally. `https://www.apache.org/dyn/closer.cgi?path=drill/drill-0.9.0/apache-drill-0.9.0.tar.gz`.

File: Library/Formula/apache-drill.rb [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+  version "0.9.0"
```
Comment (DomT4) [inline Library/Formula/apache-drill.rb]: This is redundant here. The URL is simple enough that `brew` can grab the version automatically.

**Comment by JGailor** [comment]
Thanks for the feedback @DomT4 

I made some changes in-line with your feedback.  Let me know if anything else is missing.

File: Library/Formula/apache-drill.rb [hunk 5]
```diff
@@ -0,0 +1,17 @@
+class ApacheDrill < Formula
+  homepage "https://drill.apache.org/download/"
+  url "https://www.apache.org/dyn/closer.cgi?path=drill/drill-0.9.0/apache-drill-0.9.0.tar.gz"
+  sha256 "1680b4b937d26623f0f01d7719b0809decf3341f445f5caea3feed1ecbc16c4d"
+
+
```
Comment (DomT4) [inline Library/Formula/apache-drill.rb]: Only need one blank line here, not two.

File: Library/Formula/apache-drill.rb [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+  def install
+    libexec.install Dir["*"]
+    ["drill_dumpcat", "runbit", "sqlline"].each do |bin_file|
```
Comment (DomT4) [inline Library/Formula/apache-drill.rb]: ` * 9: Trailing whitespace was found `

**Comment by JGailor** [comment]
Hopefully all set now.

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Pull request closed without merging**